app/components/user_list/__snapshots__/index.test.tsx.snap

```diff
@@ -901,6 +790,777 @@ exports[`components/channel_list_row should show results no tutorial 1`] = `
+                  },
+                  {
+                    "opacity": 1,
+                  },
+                ]
+              }
+            >
+              <View
+                style={
+                  {
+                    "borderRadius": 21.5,
+                    "height": 42,
+                    "width": 42,
+                  }
+                }
+                testID="user_list.user_item.2.profile_picture"
+              >
+                <Icon
+                  name="account-outline"
+                  size={24}
+                  style={
+                    {
+                      "color": "rgba(63,67,80,0.48)",
+                    }
+                  }
+                />
+              </View>
+            </View>
+            <View
+              style={
+                [
+                  {
+                    "flex": 1,
+                    "flexDirection": "column",
+                    "justifyContent": "center",
+                    "paddingHorizontal": 10,
+                  },
+                  {
+                    "opacity": 1,
+                  },
+                ]
+              }
+            >
+              <View
+                style={
+                  {
+                    "flexDirection": "row",
+                  }
+                }
+              >
+                <Text
+                  ellipsizeMode="tail"
+                  numberOfLines={1}
+                  style={
+                    {
+                      "color": "#3f4350",
+                      "fontFamily": "OpenSans",
+                      "fontSize": 16,
+                      "fontWeight": "400",
+                      "height": 24,
+                      "lineHeight": 24,
+                      "maxWidth": "80%",
+                    }
+                  }
+                  testID="user_list.user_item.2.display_name"
+                >
+                  rocky
+                </Text>
+              </View>
+            </View>
+            <View
+              style={
+                {
+                  "alignItems": "center",
+                  "justifyContent": "center",
+                }
+              }
+            >
+              <Icon
+                color="rgba(63,67,80,0.32)"
+                name="circle-outline"
+                size={28}
+              />
+            </View>
+          </View>
+        </View>
+      </View>
+    </View>
+    <View
+      onFocusCapture={[Function]}
+      onLayout={[Function]}
+      style={null}
+    />
+    <View
+      onLayout={[Function]}
+    >
+      <View
+        style={
+          {
+            "alignItems": "center",
+            "flex": 1,
+            "justifyContent": "center",
+          }
+        }
+      >
+        <ActivityIndicator
+          color="#1c58d9"
+          size="large"
+        />
+      </View>
+    </View>
+  </View>
+</RCTScrollView>
+`;
+
+exports[`components/channel_list_row should show results no tutorial 1`] = `
```

Comment:
A bit surprised there is a new test. Is this expected?

Reply:
I think this was a weird diff from github.  Nevertheless, I reverted to the previous tests and added a new test which tests two users with different starting alpha letters "rocky" and "john"